packit_service/events/abstract/base.py

```diff
@@ -5,16 +5,16 @@
 from typing import Optional, Union
 
 from ogr.abstract import GitProject, PullRequest
-from packit.config import PackageConfig
 
-from packit_service.config import PackageConfigGetter, ServiceConfig
+from packit_service.config import ServiceConfig
 from packit_service.models import (
     AbstractProjectObjectDbType,
     CoprBuildTargetModel,
     ProjectEventModel,
     TFTTestRunTargetModel,
     filter_most_recent_target_names_by_status,
 )
+from packit_service.package_config_getter import PackageConfig, PackageConfigGetter
```

Comment:
Why the indirect import of `PackageConfig`?

Reply:
good catch, I think this is on IDE, going to change it